packages/grafana-ui/src/components/IconButton/IconButton.tsx

```diff
@@ -61,27 +61,19 @@ export const IconButton = React.forwardRef<HTMLButtonElement, Props>(
     const styles = getStyles(theme, limitedIconSize, variant);
     const tooltipString = typeof tooltip === 'string' ? tooltip : '';
 
-    // When using tooltip, ref is forwarded to Tooltip component instead for https://github.com/grafana/grafana/issues/65632
-    const button = (
-      <button
-        ref={tooltip ? undefined : ref}
-        aria-label={ariaLabel || tooltipString}
-        {...restProps}
-        className={cx(styles.button, className)}
-      >
-        <Icon name={name} size={limitedIconSize} className={styles.icon} type={iconType} />
-      </button>
+    // ref is forwarded to Tooltip component instead for https://github.com/grafana/grafana/issues/65632
+    return (
+      <Tooltip ref={ref} content={tooltip} placement={tooltipPlacement}>
+        <button
+          ref={ref}
+          aria-label={ariaLabel || tooltipString}
```

Comment:
Is the Tooltip available to screen readers by default? Maybe that is an issue with the Tooltip component.

Reply:
@tskarhed Sorry, I'm not sure whether I understand the question. 
This line checks whether there is a dedicated string for aria-label. If there isn't it will use the tooltip or an empty string (if the tooltip is something else than a string).

Reply:
Sorry for not being clear. My point is that if Tooltip is accessible by default, do we need to explicitly have the tooltipString as a fallback for the aria-label?